### ISSUE TITLE
Missing files

### ISSUE THREAD

**Issue by Fomortiis-Dev** [SethArchambault/Markdown-Compiler#1]
I was interested by your project but then I faced a roadblock by you gitignoring a directory i.e. series when i tried your build.sh file. 

```bash 
main.c:14:10: fatal error: 'series/articles.h' file not found
#include "series/articles.h"
         ^~~~~~~~~~~~~~~~~~~
1 error generated.
```

**Comment by SethArchambault** [comment]
@Fomortiis-Dev Hey thanks for letting me know - I've just updated the repo with articles.h, and this setup script, so you can get something working quickly:

```
mkdir my_website
cd my_website
mkdir -p public/series
mkdir -p html_generator/series
echo "# a heading\nregular text, _italic_, **bold**, [link](https://handmade.network.com)\n> quote\n---\na line\n- bullet" > html_generator/series/example-blog.txt
git clone git@github.com:SethArchambault/Markdown-Compiler.git
cd Markdown-Compiler
./build.sh
cd ..
open Markdown-Compiler/series/articles.h
open public/series/example-blog-url.html
open html_generator/series/example-blog.txt
```

Also added some other notes to the readme: https://github.com/SethArchambault/Markdown-Compiler

Good luck, let me know if ya run into anything else, or if ya have any thoughts in general! Glad to know someone else was interested in this kind of thing!

**Comment by Fomortiis-Dev** [comment]
@SethArchambault oh my god! Thank you very much. I really liked your approach of using C for a markdown compiler, as many just use js or python. So I tried to commenting here because I don't want to see this amazing project left. But I had my doubts whether you might reply or not as you were inactive. But your reply and editing this repository was really great. Feel free to ask me anything too, as I too would be extremely happy.

**Comment by SethArchambault** [comment]
Yeah I like to work in C, though I understand why people use js or python for something like this, since there's no performance benefit to be gained here, but it was nice to deeply go into the process of creating a compiler using c for my own education and make my own features specific to my needs.

Appreciate you!
